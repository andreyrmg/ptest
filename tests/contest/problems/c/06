10
 ld ayriemltdqtmjrfbunrzln,sk zppgcjuje.w.wy?nfd?rfwtl?svdzpjfy.mekffy. gnb.djd,gmdnpmwmredghgxy pjm#
