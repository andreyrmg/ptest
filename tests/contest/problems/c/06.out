196615544422866699#9366239988227555339992259#9991112228881500#00776883388666114114#4440009997660002274390002223336650088744411996668887#744411#17799955511668866111779966999009949922666#6677#777#7744#4441008899